### DISCUSSION TITLE
blog/nextjs-boilerplate-2023

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#366] [General]
# blog/nextjs-boilerplate-2023

The top Next.js boilerplates from several tech stacks to match with your needs

https://theodorusclarence.com/blog/nextjs-boilerplate-2023

**Comment by mdirshaddev** [comment]
Nice article.

**Comment by PrinceAdvant** [comment]
Interesting list. It's impressive how much the Next.js starter ecosystem has evolved since 2023.

One trend I've noticed recently is the shift from feature-focused boilerplates toward production-focused SaaS foundations. Early starters were mostly about getting auth and UI running quickly, while newer projects are putting more emphasis on permissions, organisations, billing workflows, storage, and long-term maintainability.

Kostra (https://kostra.io) is one example of that newer approach. It's interesting seeing how the ecosystem is moving from simple starters toward more complete SaaS foundations.
